src/App.tsx

```diff
@@ -11,11 +11,19 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => (
+      {
+        count: state.count + 1,
+      }
+    ));
```

Comment:
```suggestion
    this.setState(state => ({
        count: state.count + 1,
      }));
```
checkList: try to solve the task in 6 lines of code :)